### DISCUSSION TITLE
Using the recipe guide to create a button recipe seems to remove some props

### DISCUSSION THREAD

**Discussion by xbohov01** [chakra-ui/chakra-ui#9339] [Q&A] [answered]
I'm testing out Chakra v3 by migrating a small application.
I'm converting my button styles to recipes following the recipe guide [here](https://www.chakra-ui.com/docs/theming/recipes).

My understanding is that to be able to use the recipe I have to (after adding it to the theme config) replace the CLI generated button snippet with the one in the guide:
```js
"use client"

import { chakra } from "@chakra-ui/react"
import { buttonRecipe } from "./button.recipe"

export const Button = chakra("button", buttonRecipe)
```

But doing so and using the newly created component seems to remove the loading and disabled props from the Button.
Also I should note that after running the CLI typegen I can use the variant I defined in my recipe so that part seems to be working. 

Am I missing something? The docs are pretty unclear about when to modify snippets or replace them or ignore them.
Thanks in advance!

**Comment by segunadebayo** [comment, marked as answer]
Do not replace `Button` with `chakra("button", recipe)`. That drops `loading`, `disabled`, and the real Button API.

Merge your recipe into `createSystem(defaultConfig, yours)` and keep importing `Button` from `@chakra-ui/react`.
